modules/database/src/template/top/exampleApp/Db/circle.db

```diff
@@ -1,3 +1,7 @@
+# SPDX-FileCopyrightText: 2017 ITER Organization
+#
+# SPDX-License-Identifier: EPICS
+
```

Comment:
@ralphlange Would this be the correct copyright attribution for this file? You added it to Git in 1c03e615f16ab9a1279456846b414858a4e94274 but I suspect the original might have had a different source.

Reply:
Yes, that's fine.